pkg/compilers/osv/osv_aws.go

```diff
@@ -1,23 +1,36 @@
 package osv
 
 import (
+	"io/ioutil"
+	"os"
+
 	"github.com/emc-advanced-dev/pkg/errors"
 	"github.com/emc-advanced-dev/unik/pkg/types"
 )
 
-type OsvAwsCompiler struct {
-	OSvCompilerBase
-}
-
 const OSV_AWS_MEMORY = 1024
 
-func (osvCompiler *OsvAwsCompiler) CompileRawImage(params types.CompileImageParams) (_ *types.RawImage, err error) {
-	resultFile, err := osvCompiler.CreateImage(params, true)
+type AwsCompilerHelper struct {
+	CompilerHelperBase
+}
+
+func (b *AwsCompilerHelper) Convert(params ConvertParams) (*types.RawImage, error) {
+	// Convert to WMDK format.
+	resultFile, err := ioutil.TempFile("", "osv-boot.vmdk.")
```

Comment:
This conversion is repeated in all helpers. I suggest you add it to the base helper (if so, ignore the embed comment above). Alternatively, you can add a utility function because the helper is not used in this scope (lines 18-30 in AWS).